content/english/android-tictactoe/getting-started.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Getting Started"
+date: 2020-07-27T14:17:07.42-07:00
+draft: true
+weight: 1
+---
+
+# Getting Started
+
+## What is Android?
+Android is currently the most used mobile operating system in the world! Android Studio lets you create apps that can be used by these Android devices. In fact, if you have an Android phone, then you can even download the app you’re making today onto your phone!
+
+## Download the Project Files
+[Download](../resources/_gen/files/TicTacToe.zip) the project files for your game. On this adventure, you will need to solve the puzzles marked with `PUZZLE` tags to complete the game.
+
+## Start Android Studio
+Run Android Studio and open the project. Your view will look similar to this:
```

Comment:
A step-by-step walkthrough for how to open Android Studio and open the files would be helpful.

Reply:
I can add details for a Windows box, but won't be able to create a walkthrough for MacOS

Reply:
That makes sense. I think for now you can assume Windows (and we can go back and add Mac instructions later).